pulpcore/pulpcore/app/viewsets/repository.py

```diff
@@ -128,6 +129,87 @@ class Meta:
         fields = ContentAdaptorFilter.Meta.fields
 
 
+class RepositoryVersionContentInFilter(CharInFilter):
+    """
+    Filter used to get the repository versions where some given content can be found.
+
+    Given a list of content_id, this filter will:
+        1. Get the RepositoryContent that the id can be found in
+        2. Get a list of version_added and version_removed where the content was
+           changed on the repository
+        3. Calculate and return the versions that the contents can be found on
+    """
+
+    def filter(self, qs, value):
+        """
+        Args:
+            qs (django.db.models.query.QuerySe): The RepositoryVersion Queryset
+            value (list): of content hrefs to filter
+
+        Returns:
+            Queryset of the RepositoryVersions containing specified content
+        """
+
+        # If no value is passed, return queryset as-is
+        if not value:
+            return qs
+
+        contend_pks = [RepositoryVersionViewSet.get_content_unit_pk(url) for url in value]
```

Comment:
[A PR just got merged that affects this](https://github.com/pulp/pulp/pull/3311). I think you just need this now:

```python
contend_pks = [GenericNamedModelViewSet.get_resource(url, Content).pk for url in value]
```